doc/en/example/multipython.py

```diff
@@ -30,8 +30,8 @@ def __init__(self, version, picklefile):
         self.picklefile = picklefile
 
     def dumps(self, obj):
-        dumpfile = self.picklefile.dirpath("dump.py")
-        dumpfile.write(
+        dumpfile = self.picklefile.parent / "dump.py"
```

Comment:
lets use `Path.with_name` for those cases instead of `path.parent / "newname"`

Reply:
Right, changed.